README.md

```diff
@@ -1,57 +1,43 @@
-**Please ensure you work through the tickets in numerical order.**
+After successfully creating the required environment variables on the root folder. Please run the following commands to create both the test & development databases:
+> ```npm run setup-dbs```
 
-## Git Branching and Pull Requests
+> ``` npm run seed```
 
-You will be working on each ticket on a new **branch**.
+With this, you will have now complete access to the repostitory. You will be able to navigate below to the available endpoints and test the available endpoints locally using Jest.
 
-To create and switch to a new git branch use the command:
+## Testing Endpoints With Jest & Localhost
+This project has been created with the core Agile practice of TDD (Test-Driven-Development) using the popular Jest Testing framework to ensure that the API works as it is intended to.
 
-```
-git checkout -b <new branch name>
-```
+Please enter the following command to view the tests that have been created in mind for this project, and feel free to both test & add/remove tests incrementally on your own local copy of this repository:
+> ```npm test```
 
-This will create a branch and move over to that branch. (Omit the `-b` flag if you wish to switch to an already existing branch).
-
-We recommend that you name the branch after the number assigned to each ticket via the header. eg. `ncnews-1`
-
-When pushing the branch to git hub ensure that you make reference to the branch you are pushing to on the remote.
-
-```
-git push origin <branch name>
-```
-
-From github you can make a pull request and share the link and ticket number via a pull request specific nchelp using the command `nchelp pr`. A tutor will swing by to review your code. Ensure that you keep your trello up to date whilst you await the PR approval. Regular `nchelp` will be available for when you need support.
-
-Once a pull request been accepted be sure to switch back to the main branch and pull down the updated changes.
-
-```
-git checkout main
-
-git pull origin main
-```
-
-You can tidy up your local branches once they have been pull into main by deleting them:
-
-```
-git branch -D <local branch>
-```
-
-## Husky
-
-To ensure we are not commiting broken code this project makes use of git hooks. Git hooks are scripts triggered during certain events in the git lifecycle. Husky is a popular package which allows us to set up and maintain these scripts. This project makes use a _pre-commit hook_. When we attempt to commit our work, the script defined in the `pre-commit` file will run. If any of our tests fail than the commit will be aborted.
-
-The [Husky documentation](https://typicode.github.io/husky/#/) explains how to configure Husky for your own project as well as creating your own custom hooks.\_
+Lastly, if you would like to access the API on your localhost upon successfully setting up this API, please run this following command and navigate to the endpoints available below:
+> ```npm run start```
+## Available Endpoints
```

Comment:
This is really nice attention to detail - although word of warning, there is a ticket specifically handling the communication of which endpoints are available toward the end of the sprint. It's nice to see for now though :)

Reply:
Thank you for the quick response, Rose! I didn't see that ticket but thank you for informing me about it, I'll be sure to take care and update it as needed 😁